TCC.Lib/Benchmark/BenchmarkHelper.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using TCC.Lib.Blocks;
+using TCC.Lib.Options;
+
+namespace TCC.Lib.Benchmark
+{
+    public class BenchmarkHelper
+    {
+        private readonly CancellationTokenSource _cancellationTokenSource;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the field '_cancellationTokenSource' and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1450)